fancy-explosions/BangazonSite
# Created shopping cart view along with delete links for each product in the cart

# Description

View cart nav link now shows each item in the users shopping cart and allows them to delete any product in the list that they want.


Fixes Issue # (link issue ticket here)

## Type of change

Please delete options that are not relevant.




# Testing Instructions
Git fetch --all
git checkout cm-shoppingCart
Click view cart link on nav bar and it should show each product in your shopping cart and also allow you to delete any item in the cart that you want.



# Checklist:
